utils/rector/config/foundry-set.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\FuncCall\RenameFunctionRector;
+use Rector\Renaming\Rector\MethodCall\RenameMethodRector;
+use Rector\Renaming\ValueObject\MethodCallRename;
+use Zenstruck\Foundry\ModelFactory;
+use Zenstruck\Foundry\Object\ObjectFactory;
+use Zenstruck\Foundry\Persistence\PersistentObjectFactory;
+use Zenstruck\Foundry\Persistence\PersistentProxyObjectFactory;
+use Zenstruck\Foundry\Persistence\Proxy;
+use Zenstruck\Foundry\Proxy as LegacyProxy;
+use Zenstruck\Foundry\Utils\Rector\AddProxyToFactoryCollectionTypeInPhpDoc;
+use Zenstruck\Foundry\Utils\Rector\ChangeDisableEnablePersist;
+use Zenstruck\Foundry\Utils\Rector\ChangeFactoryBaseClass;
+use Zenstruck\Foundry\Utils\Rector\ChangeFactoryMethodCalls;
+use Zenstruck\Foundry\Utils\Rector\ChangeInstantiatorMethodCalls;
+use Zenstruck\Foundry\Utils\Rector\ChangeLegacyClassImports;
+use Zenstruck\Foundry\Utils\Rector\PersistenceResolver;
+use Zenstruck\Foundry\Utils\Rector\RemoveProxyRealObjectMethodCallsForNotProxifiedObjects;
+
+return static function (RectorConfig $rectorConfig): void {
+    /**
+     * This must be overridden in user's `rector.php` to provide a path to the object manager
+     * @see https://github.com/phpstan/phpstan-doctrine#configuration
+     */
+    $rectorConfig->singleton(PersistenceResolver::class);
+
+//    $rectorConfig->when(ObjectMetadataResolver::class)
+//        ->needs('$objectManagerLoader')
+//        ->giveConfig('%doctrine.objectManagerLoader%'); // this does not work
```

Comment:
https://github.com/rectorphp/rector/discussions/8379

Reply:
actually it is more complex from user's POV to leverage phpstan's configuration than to override the service definition

see  https://github.com/zenstruck/foundry/pull/544/files#diff-57df83b36e5e3d69ad38354b8047370ad4841d60ce901ee2d5020a3a28784a02